src/configurations/destinations/zoho_dev/db-config.json

```diff
@@ -10,14 +10,15 @@
     "transformAtV1": "router",
     "saveDestinationResponse": true,
     "excludeKeys": [],
-    "supportedSourceTypes": ["warehouse"],
+    "supportedSourceTypes": ["warehouse", "cloud"],
```

Comment:
what about `web` etc ?

Reply:
This destination generally supports warehouse only source with record event type. But we are adding this source support for testing purposes. If we need to test with any other sources will create a pr.